src/test/kotlin/org/prebid/cache/functional/testcontainers/container/PrebidCacheContainer.kt

```diff
@@ -0,0 +1,58 @@
+package org.prebid.cache.functional.testcontainers.container
+
+import org.prebid.cache.functional.testcontainers.ContainerDependencies
+import org.testcontainers.containers.GenericContainer
+
+class PrebidCacheContainer(imageName: String, config: Map<String, String>) :
+        GenericContainer<PrebidCacheContainer>(imageName) {
+
+    companion object {
+        private const val PORT = 8080
+        private const val DEBUG_PORT = 8000
+        private const val FIXED_EXPOSED_APPLICATION_PORT = 49100
+        private const val FIXED_EXPOSED_DEBUG_PORT = 49101
+
+        private val USE_FIXED_PORTS = (System.getProperty("useFixedContainerPorts") ?: "false").toBoolean()
+    }
+
+    init {
+        withNetwork(ContainerDependencies.network)
+        withExposedPorts(PORT, DEBUG_PORT)
+        withFixedExposedPorts()
+        withDebug()
+        withConfig(config)
+    }
+
+    fun getHostUri(): String {
+        return "http://$host:${getHostPort()}"
+    }
```

Comment:
Consider using` = result` instead of old Java style `{ return result; }`. It's easier to read and more Kotlin way. Also, please, check for similar occurences. After this refactoring, most of methods will become one-liners.